tensorflow/compiler/mlir/tensorflow/ir/tf_ops_a_m.cc

```diff
@@ -339,15 +339,18 @@ void BatchToSpaceOp::getCanonicalizationPatterns(
 //   are not unknown.
 //
 static LogicalResult Verify(BiasAddOp op) {
-  StringRef format = op.data_format();
-  if (format == "NHWC") {
+  std::string data_format = op.data_format().str();
```

Comment:
optional nit: To avoid materializing the string here for FormatFromString, let's change input type of the function to be string_view. Then, we can convert from StringRef to string_view directly.